hack/apidocs.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/go-openapi/loads"
+	"github.com/openshift/origin/pkg/apidocs"
+)
+
+func writeAPIDocs(root string) error {
+	err := os.RemoveAll(root)
+	if err != nil {
+		return err
+	}
+
+	doc, err := loads.JSONSpec("api/swagger-spec/openshift-openapi-spec.json")
```

Comment:
from a file instead of the server?  What does the file combine (for those who don't recall)